doc/specs/Configuration object for profiles.md

```diff
@@ -0,0 +1,132 @@
+defined in a `Profile` can be defined in this new object (for example, we do not want parameters which
+would cause a resize in this object.) Here is the list of parameters we will allow:
+
+- Anything regarding colors: `colorScheme`, `foreground`, `background`, `cursorColor` etc
+- Anything regarding background image: `path`, `opacity`, `alignment`, `stretchMode`
+- `cursorShape`
+
+We may wish to allow further parameters in these objects in the future (like `bellStyle`?). The addition
+of further parameters can be discussed in the future and is out of scope for this spec.
+
+### Inheritance
+
+In the case that not all of the allowed parameters are defined in an appearance object, we will obtain the
+values for those parameters in the following matter:
+
+If the profile defines an `unfocusedConfig`, any parameters not explicitly defined within it will adopt
+the values from the profile itself. If the profile does not define an `unfocusedConfig`, then the global/default `unfocusedConfig` is used
+for this profile.
+
+Thus, if a user wishes for the unfocused state to look the same as the focused state for a particular profile,
+while still having a global/default unfocused state appearance, they simply need to define an empty `unfocusedConfig`
+for that profile (similarly, they could define just 1 or 2 parameters if they wish for minimal changes between the focused
+and unfocused states for that profile). If they do not define any `unfocusedConfig` for the profile, then
+the global/default one will be used.
+
+If neither the profile nor the global settings have defined an `unfocusedConfig`, then there will be no
+appearance change when switching between focused and unfocused states.
+
+## UI/UX Design
+
+Users will be able to add a new setting to their profiles that will look like this:
+
+```
+"unfocusedConfig":
+{
+    "colorScheme": "Campbell",
+    "cursorColor": "#888",
+    "cursorShape": "emptyBox",
+    "foreground": "#C0C0C0",
+    "background": "#000000"
+}
+```
+
+## Capabilities
+
+### Accessibility
+
+Does not affect accessibility.
+
+### Security
+
+Does not affect security.
+
+### Reliability
+
+This is another location in the settings where parsing/loading the settings may fail. However, this is the case
+for any new setting we add so I would say that this is a reasonable cost for this feature.
+
+### Compatibility
+
+Should not affect compatibility.
+
+### Performance, Power, and Efficiency
+
+## Potential Issues
+
+Inactive tabs will be 'rendered' in the background with the `UnfocusedRenderingParams` object, we need to make
+sure that switching to an inactive tab (and so causing the renderer to update with the 'normal' parameters)
+does not cause the window to flash/show a jarring indicator that the rendering values changed.
+
+When certain appearance settings are changed via OSC sequences (such as the background color), only the focused/regular

```

Comment:
I think there's a issue laying around about "make all the settings properties getters-only", and that touches on this subject. The settings should really only be read-only, but TermControl is kinda also using them as the bucket for it's runtime appearance as well. So it needs to kinda be applied as 
```
runtimeAppearance 
	+ (unfocused? unfocusedAppearance : defaultAppearance)
```

Reply:
Just to clarify, you're saying that if we get an OSC sequence to change some appearance value, we should apply it to the unfocused appearance if the control is not focused when the OSC is processed?

Reply:
Oh derp. No, sorry for confusing you. I meant `appearance = focused? default + runtime changes : unfocused`

So I guess I'm imagining that if someone changes the BG color with an OSC< that only changes the _focused_ BG color, _never_ the unfocused one.

Plus, this way, when the settings reload, the runtime changes would still persist. This would apply for things like changing the font size too (though that's not as relevant for this spec)

Reply:
> So I guess I'm imagining that if someone changes the BG color with an OSC< that only changes the focused BG color, never the unfocused one.

Makes sense, I'll add that in!